webpack/components/AnsibleRoles/AnsibleRolesPage.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import TableIndexPage from 'foremanReact/components/PF4/TableIndexPage/TableIndexPage';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { getDocsURL } from 'foremanReact/common/helpers';
+import { AnsibleRolesImportButton } from './components/AnsibleRolesImportButton';
+import { getActions } from './AnsibleRolesPageHelpers';
+
+export const AnsibleRolesPage = () => (
+  <TableIndexPage
+    header={__('Ansible Roles')}
+    controller="ansible_roles"
+    apiUrl="/ansible/ui_ansible_roles"
+    apiOptions={{ key: 'ANSIBLE_ROLES_API_REQUEST_KEY' }}
+    hasHelpPage
+    creatable={false}
+    isDeleteable
+    searchable
+    customToolbarItems={<AnsibleRolesImportButton />}
```

Comment:
Should only be visible if a user has a `import_ansible_roles` permission

Reply:
Good idea, fixed.